### ISSUE TITLE
Unnecessary aria-label on text links opening new windows

### ISSUE THREAD

**Issue by aardrian** [WordPress/gutenberg#20129]
**Describe the bug**
If I create a link that opens in a new window, the link gets an `aria-label` attribute with the full text of the link _plus_ the text "(opens in a new tab)".

For pages that are machine-translated, `aria-label` is only automatically translated in Chrome and ChromiEdge. That means for more than half of screen reader users who visit a page in another language, they will hear the link in the page's original language even though the link text itself will be translated.

This is an accessibility / internationalization issue.

**To reproduce**
1. Go to https://wordpress.org/gutenberg/.
2. Select any piece of text and add a link.
3. After adding the link, toggle the option to _open in new tab_.
4. Change to _Edit as HTML_.

**Expected behavior**
The `aria-label` attribute should not be used.

Instead of this:
```
<a href="https://example.com" target="_blank" rel="noreferrer noopener" aria-label="building engaging content (opens in a new tab)">building engaging content</a>
```

It should be this:
```
<a href="https://example.com" target="_blank" rel="noreferrer noopener">building engaging content</a>
```

Gutenberg can add plain text into the link text announcing the new window, thereby benefiting all users regardless of language.

**Screenshots**
![Text link highlighted with new window chosen.](https://user-images.githubusercontent.com/1376607/74111646-44b8dc80-4b64-11ea-9448-9a9017a18640.png)
![HTML view showing the aria-label on the link.](https://user-images.githubusercontent.com/1376607/74111696-9f523880-4b64-11ea-8219-b7bd83c2e00b.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox 72, Chrome 79

**Additional context**
* Per the latest [WebAIM Screen Reader Survey Results (September 2019)](https://webaim.org/projects/screenreadersurvey8/#browsers), less than half of screen reader users use Chrome as their primary browser.

**Comment by swissspidy** [comment]
Sounds like a duplicate of #18727.

**Comment by aardrian** [comment]
Similar problem with same solution.

In Marco's case, he is an author trying to add a link in the language of the site and getting the "opens in new window" text in English. That affects site authors.

In my example, a user would try to auto-translate a page, and the entire text string (not the just "opens in new window") would not get translated.

Thanks for linking that, though. I did not find it when I looked for existing issues.

**Comment by youknowriad** [comment]
also, duplicate of #12325 which is also discussed in #18742
